cpp/log/cert.cc

```diff
@@ -546,12 +588,311 @@ Cert::Status Cert::ExtensionStructure(int extension_nid,
       LOG(WARNING) << "Corrupt extension data";
       LOG_OPENSSL_ERRORS(WARNING);
     }
+
     return FALSE;
   }
 
   return TRUE;
 }
 
+bool IsRedactedHost(const string& hostname) {
```

Comment:
Pop these into an anonymous namespace (unless you need access to them from other places), that way the compiler will warn if they're unused at any point.